patroni/ctl.py

```diff
@@ -454,7 +454,7 @@ def parse_scheduled(scheduled):
             raise PatroniCtlException(message.format(scheduled))
         return scheduled_at
 
-    return None
```

Comment:
Once again:

If we want to failover immediately, we should set `scheduled_at` to `None`

PERIOD

That is how Patroni works. If you set it to the current time (now) it might be DISCARDED due to time difference between the host where you run patronictl and the host where the leader runs.